pact4s/README.md

```diff
@@ -0,0 +1,50 @@
+# pact4s [Under construction]
+
+pact4s is used for contract testing.
+
+# Dependencies
+
+```scala
+  val pact4sDependencies = Seq(
+    pact4sScalaTest,
+    pact4sCirce,
+    http4sEmberClient,
+    http4sDsl,
+    http4sEmberServer,
+    http4sCirce,
+    circeCore,
+    typelevelCat,
+    scalaTest
+  )
+
+lazy val pact4s = project.in(file("pact4s"))
+  .settings(pact4sSettings)
+  .dependsOn(http % "test->test;compile->compile")
+```
+
+## Building and running contract tests
+Clone the repo.
+```
+$ git clone https://github.com/DataBiosphere/leonardo.git 
+$ cd leonardo
+```
+
+If you are already using OpenJDK 17, run the following command. 
+```
+$ sbt "project pact4s" clean test  
+```
+
+Otherwise, you can run the command inside a docker container with OpenJDK 17 installed. 
+```
+docker run --rm -v $PWD:/working \
+                -v jar-cache:/root/.ivy \
+                -v jar-cache:/root/.ivy2 \
+                -w /working \
+                sbtscala/scala-sbt:openjdk-17.0.2_1.7.2_2.13.10 \
+                sbt "project pact4s" clean test
+```
```

Comment:
I think we can assume most ppl have java 17 installed...so probably just say `sbt "project pact4s" clean test`?